README.md

```diff
@@ -1,2 +1,68 @@
-# portfolio
-My Portfolio Project built following Microverse project standards
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio Website
+
+> This is a website i am building to show case my portfolio. From time to time i will be making changes and updating it.
+
+
+## Built With
+
+- HTML,CSS
+- No frameworks used yet
+- Linters are used in this projects
+
+## Live Demo (if available)
+
+[Live Demo Link](https://nolivedemoyet.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+I recommend you install node Js in your local environment
+### Setup
+No need for any set up just clone the project and run
+### Install
+No need for any set up just clone the project and run
+
+### Usage
+NA
+### Run tests
+so when you make a pull request there will be some tests that your code should pass ensure that your code passes all of the test.
+### Deployment
+NA
+
+
+## Authors
```

Comment:
- [ ] Kindly change the `Authors` to `Author` as this is a solo project which is supposed to carry single author.